styles.css

```diff
@@ -0,0 +1,160 @@
+* {
+  font-family: poppins, sans-serif;
+  color: #172b4d;
+}
+
+.head {
+  width: 76px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 22px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  margin: 15px;
+}
+
+.h {
+  width: 327px;
+  height: 150px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding-top: 114px;
+  margin-left: 10px;
+}
+
+.info1 {
+  width: 327px;
```

Comment:
- [Required] Please consider removing the fixed width on the `.info1` element this is because your design for mobile devices is supposed to be responsive across all devices including large screens.